pkgs/development/tools/devpod/default.nix

```
@@ -0,0 +1,185 @@
+{ lib
+, buildGoModule
+, copyDesktopItems
+, darwin
+, desktopToDarwinBundle
+, fetchFromGitHub
+, fetchYarnDeps
+, gtk3
+, installShellFiles
+, jq
+, libayatana-appindicator
+, libsoup
+, makeDesktopItem
+, mkYarnPackage
+, openssl
+, pkg-config
+, rust
+, rustPlatform
+, stdenv
+, testers
+, webkitgtk
+}:
+
+let
+  pname = "devpod";
+  version = "0.3.7";
+
+  src = fetchFromGitHub {
+    owner = "loft-sh";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-s5O7pcrynfwgye6NdcF6XOCRDf1VS9i8Z3ghOQnjiKg=";
+  };
+
+  meta = with lib; {
+    description = "Codespaces but open-source, client-only and unopinionated: Works with any IDE and lets you use any cloud, kubernetes or just localhost docker. ";
```

```suggestion
    description = "Codespaces but open-source, client-only and unopinionated: Works with any IDE and lets you use any cloud, kubernetes or just localhost docker";
```

No trailing space nor dot.